config/logstash.yml

```diff
@@ -214,8 +214,8 @@
 #
 # queue.type: memory
 #
-# If using queue.type: persisted, the directory path where the data files will be stored.
-# Default is path.data/queue
+# If using queue.type: persisted, the directory path where the pipeline data files will be stored.
+# Default is path.data/queue and Logstash creates `pipeline.id` folder for each pipeline.
```

Comment:
```suggestion
# If queue.type: persisted, the directory path where the pipeline data files will be stored.  Logstash creates `pipeline.id` folder for each pipeline
# Default is path.data/queue.
```

Reply:
Please consider moving the line about Logstash creating a folder for each pipeline to the first section.